Description: missing CPPFLAGS, fix hardening-no-fortify-functions.
Index: mupen64plus-libretro-master/Makefile
===================================================================
--- mupen64plus-libretro-master.orig/Makefile	2016-02-07 11:39:40.000000000 -0200
+++ mupen64plus-libretro-master/Makefile	2016-06-12 22:28:13.334535093 -0300
@@ -402,10 +402,10 @@
 	$(CC_AS) $(CFLAGS) -c $< -o $@
 
 %.o: %.c
-	$(CC) $(CFLAGS) -c $< -o $@
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $< -o $@
 
 %.o: %.cpp
-	$(CXX) $(CXXFLAGS) -c $< -o $@
+	$(CXX) $(CPPFLAGS) $(CXXFLAGS) -c $< -o $@
 
 
 clean:
